docs/operator-manual/notifications/catalog.md

```diff
@@ -1,4 +1,12 @@
 # Triggers and Templates Catalog
+## Getting Started
+
+* Install Triggers and Templates from the catalog
```

Comment:
You need to change this inside hack/gen-catalog/main.go 
And after execute `make notification-catalog` to generate this md file

Reply:
Thank you for pointing out this was generated!  I've gone ahead and made changes to `hack/gen-catalog/main.go`. If you'd like to see a multi-line style I can try. Please let me know if you prefer a different style. 

An aside -- I had to run `go run ./hack/gen-catalog docs` to generate the docs which was part of the `make notification-docs` target. 

Original  commit amended.